Add_Getting_Started_content

Sergiodero opened the pull request:
This change adds new content for multiple pages in the Getting Started section:

- What is GTFS? (including new images added in assets)
- Why use GTFS?
- Create (including new images added in assets)
- Validate
- Publish

@isabelle-dr can you take a look?

Sergiodero committed:
Add_Getting_Started_content

This change adds new content for multiple pages in the Getting Started section

Sergiodero committed:
Fix grammar

Fix grammar on getting started pages (except Features)

isabelle-dr commented on docs/getting_started/create.md:
```diff
@@ -0,0 +1,46 @@
+
+<img class="center" width="560" height="100%" src="../../../assets/create_001.png">
+
+Each file consists of a list of multiple entries (lines of data) with several information fields. For example, each line listed in [routes.txt](/documentation/schedule/reference/#routestxt) represents a public transport route and its fields describe multiple elements of that route, like its name, description, operating agency, etc.
```
These are called "records" in the GTFS Reference (ref [Term Definitions](https://gtfs.org/schedule/reference/#term-definitions)), I recommend this term appears somewhere here.

isabelle-dr replied:
I think we can simply replace "entries" by "records"

isabelle-dr commented on docs/getting_started/create.md:
```diff
@@ -0,0 +1,46 @@
+
+<img class="center" width="560" height="100%" src="../../../assets/create_002.png">
+
+The base files for a GTFS dataset can be described as follows: A GTFS schedule dataset has one or more routes, each route ([routes.txt](/documentation/schedule/reference/#routestxt)) has one or more trips ([trips.txt](/documentation/schedule/reference/#tripstxt)), each trip visits a series of stops ([stops.txt](/documentation/schedule/reference/#stopstxt)) at specified times ([stop_times.txt](/do ...
```
```suggestion
The base files for a GTFS dataset can be described as follows: A GTFS schedule dataset has one or more routes ([routes.txt](/documentation/schedule/reference/#routestxt)), each route has one or more trips ([trips.txt](/documentation/schedule/reference/#tripstxt)), each trip visits a series of stops ([stops.txt](/documentation/schedule/reference/#stopstxt)) at specified times ([stop_times.txt](/documentation/schedule/reference/#stop_timestxt)). Trips and stop times only contain time of day information; the calendar is used to determine on which days a given trip runs ([calendar.txt](/documentation/schedule/reference/#calendartxt) and [calendar_dates.txt](/documentation/schedule/reference/#calendar_datestxt)). Additionally, multiple agencies ([agency.txt](/documentation/schedule/reference/#agencytxt)) can operate multiple routes. These files are linked to each other with fields that are cross-referenced between them.
```

isabelle-dr commented on docs/getting_started/create.md:
```diff
@@ -0,0 +1,46 @@
+- [pathways.txt](/documentation/schedule/reference/#pathwaystxt) which provides information that makes it possible to generate directions to help users navigate stations, 
+- [frequencies.txt](/documentation/schedule/reference/#frequenciestxt) which provides an alternative way to specify stop times. 
+
+For more information on all the GTFS functionalities that can be enabled, see the “what can GTFS do?” section. 
```
Add a link to the section

isabelle-dr commented on docs/getting_started/publish.md:
```diff
@@ -0,0 +1,69 @@
+
+### Feed Aggregators
+
+Sharing your GTFS data also allows it to be indexed by GTFS feed aggregation platforms, which can include state- or region-level directories of GTFS feeds, as well as international feed aggregators like the [Mobility Database](https://database.mobilitydata.org/) and [Transitland](https://www.transit.land/) (see more feed aggregators here). Being indexed on a feed aggregator increases your agency’ ...
```
```suggestion
Sharing your GTFS data also allows it to be indexed by GTFS feed aggregation platforms, which can include state- or region-level directories of GTFS feeds, as well as international feed aggregators like the [Mobility Database](https://database.mobilitydata.org/) and [Transitland](https://www.transit.land/) (see more feed aggregators [here](/resources/data). Being indexed on a feed aggregator increases your agency’s visibility and allows developers, researchers, and other interested parties to easily access your agency’s data for a variety of purposes, including the development of new applications.
```

isabelle-dr commented on docs/getting_started/publish.md:
```diff
@@ -0,0 +1,69 @@
+
+### Integration with GIS, Analysis, and other Platforms and Tools
+
+GTFS data can also be imported into and used on a variety of geospatial analysis platforms. Geographic Information Systems (GIS) programs like Esri’s ArcGIS, as well as the open-source QGIS have their own plugins and extensions that can import and visualize GTFS stop and route data.
```
```suggestion
GTFS data can also be imported and used on a variety of geospatial analysis platforms. Geographic Information Systems (GIS) programs like Esri’s ArcGIS, as well as the open-source QGIS have their own plugins and extensions that can import and visualize GTFS stop and route data.
```

isabelle-dr commented on docs/getting_started/validate.md:
```diff
@@ -0,0 +1,47 @@
+
+## Using official validators 
+
+There is a lack of consensus among stakeholders regarding what constitutes a valid dataset file. This results in data quality issues that are not addressed consistently across different organizations, as different stakeholders may have varying definitions and interpretations of what constitutes a violation. 
```
```Suggestion
Official GTFS Validators assess the quality of a dataset against the official specification, ensuring a common understanding within the industry of what constitutes a high-quality dataset. 
```

isabelle-dr replied:
Starting with "There is a lack of consensus" is a bit negative, I tried having something more neutral/factual statement here.

Sergiodero replied:
This is way better, thanks!

isabelle-dr commented on docs/getting_started/validate.md:
```diff
@@ -0,0 +1,47 @@
+
+There is a lack of consensus among stakeholders regarding what constitutes a valid dataset file. This results in data quality issues that are not addressed consistently across different organizations, as different stakeholders may have varying definitions and interpretations of what constitutes a violation. 
+
+The free and open-source canonical [GTFS Schedule validator](https://gtfs-validator.mobilitydata.org/)[^1] maintained by MobilityData ensures your GTFS data is compliant with the official GTFS Schedule Reference and Best Practices. It provides an easy-to-use validation report in an HTML format that can be opened in the browser and shared with other parties.
```
```suggestion
The free and open-source [Canonical GTFS Schedule validator](https://gtfs-validator.mobilitydata.org/)[^1] maintained by [MobilityData](https://mobilitydata.org/) ensures your GTFS data is compliant with the official [GTFS Schedule Reference](/documentation/schedule/reference/) and [Best Practices](/documentation/schedule/best-practices/). It provides an easy-to-use report that can be shared with other parties and comprehensive documentation.
```

isabelle-dr replied:
Add links and remove some of the tech jargon (HTML format)

isabelle-dr commented on docs/getting_started/validate.md:
```diff
@@ -0,0 +1,47 @@
+
+## Up-to-date data
+
+Having out-of-date data is almost worse than having no feed at all. It's not enough to simply publish information—it has to match what the rider sees and experiences. Some of the largest transit agencies update their GTFS weekly, or even daily, but most agencies will need to update their GTFS every few months, or a few times a year when service changes. This includes things like new routes or sto ...
```
```suggestion
Out-of-date data can be more problematic than having no feed available. It's not enough to simply publish information—it has to match what the rider sees and experiences. Some of the largest transit agencies update their GTFS weekly, or even daily, but most agencies will need to update their GTFS every few months, or a few times a year when service changes. This includes things like new routes or stops, timetable changes, or updates to fare structure.
```

isabelle-dr replied:
Small attempt to make the tone less negative

isabelle-dr commented on docs/getting_started/validate.md:
```diff
@@ -0,0 +1,47 @@
+
+The free and open-source canonical [GTFS Schedule validator](https://gtfs-validator.mobilitydata.org/)[^1] maintained by MobilityData ensures your GTFS data is compliant with the official GTFS Schedule Reference and Best Practices. It provides an easy-to-use validation report in an HTML format that can be opened in the browser and shared with other parties.
+
+### Usage
```
I suggest we remove this sub-title. 
If we were to add sub-titles here, having "GTFS Schedule" and "GTFS Realtime" would probably be more appropriate.

isabelle-dr commented on docs/getting_started/validate.md:
```diff
@@ -0,0 +1,47 @@
+    </div>
+</div>
+
+Similarly, using the free and open-source canonical [GTFS Realtime Validator](https://github.com/MobilityData/gtfs-realtime-validator) ensures your GTFS data is compliant with the official GTFS [Realtime Reference](/documentation/Realtime/reference.md) and [Best Practices](/documentation/Realtime/best practices.md).
```
```suggestion
Similarly, using the free and open-source canonical [GTFS Realtime Validator](https://github.com/MobilityData/gtfs-realtime-validator) ensures your GTFS data is compliant with the official [GTFS Realtime Reference](/documentation/Realtime/reference/) and [Best Practices](/documentation/Realtime/best practices/).
```

isabelle-dr commented on docs/getting_started/what_is_GTFS.md:
```diff
@@ -1,13 +1,30 @@
-GTFS is supported around the world and its use, importance, and scope has been increasing. It’s likely that an agency you know already uses GTFS to represent routes, schedule, stop locations, and other information, and that riders are already consuming it via various applications.
+Today, GTFS is the go-to [Open Standard](https://www.interoperablemobility.org/definitions/#open_standard) for thousands of public transport providers worldwide. Some agencies produce this data themselves, while others employ a vendor to create and maintain data for them. 
+
+## Two specifications in one
```
Maybe a more self explanatory "Support for static and dynamic data"?

isabelle-dr commented on docs/getting_started/what_is_GTFS.md:
```diff
@@ -1,13 +1,30 @@
+
+GTFS Schedule contains information about routes, schedules, fares, and geographic transit details among many other features, and it is presented in simple text files[^1]. This straightforward format allows for easy creation and maintenance without relying on complex or proprietary software.
+
+GTFS Realtime contains trip updates, vehicle positions, and service alerts, using a different format based on [Protocol Buffers](https://developers.google.com/protocol-buffers/). This part of GTFS works in conjunction with GTFS Schedule in order to inform passengers of service disruptions and updated arrival times.
```
```suggestion
GTFS Realtime contains trip updates, vehicle positions, and service alerts, using the [Protocol Buffers](https://developers.google.com/protocol-buffers/) format. This part of GTFS works in conjunction with GTFS Schedule in order to inform passengers of service disruptions and updated arrival times.
```

isabelle-dr replied:
small suggestion to lighten the text

isabelle-dr commented on docs/getting_started/what_is_GTFS.md:
```diff
@@ -1,13 +1,30 @@
+
+GTFS Realtime contains trip updates, vehicle positions, and service alerts, using a different format based on [Protocol Buffers](https://developers.google.com/protocol-buffers/). This part of GTFS works in conjunction with GTFS Schedule in order to inform passengers of service disruptions and updated arrival times.
+
+The detailed information for both specifications is hosted in documents known as the Reference documents. An updated copy of this information can be found in the [Technical Documentation section](/documentation/overview).
```
I think the term "Reference" under the "Technical documentation" section speaks for itself. I wouldn't call this a copy.

```suggestion
The GTFS Schedule and GTFS Realtime reference documentation is available in the [Technical Documentation section](/documentation/overview).
```

isabelle-dr commented on docs/getting_started/what_is_GTFS.md:
```diff
@@ -1,13 +1,30 @@
+
+<a href="https://www.flaticon.com/authors/freepik" title="Icons by Freepik">Icons created by Freepik - Flaticon</a>
+
+[^1]: In addition to text files, the GeoJSON format is now also supported in GTFS to represent certain elements of Demand-responsive services.
```
```suggestion
[^1]: In addition to text files, the [GeoJSON](https://geojson.org/) format is now also supported in GTFS to represent certain elements of Demand-responsive services.

isabelle-dr commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,17 +1,39 @@
-GTFS is used by over 10,000 transit agencies in over 100 countries. Most transit agencies have heard of GTFS, and it has quickly become an industry standard. Some agencies produce this data themselves, while others employ a vendor to create and maintain data for them. And because it is a simple, text-based Open Standard, many transit technology vendors can already read and write to GTFS files. By ...
+Over 10,000 transit agencies in 100+ countries rely on GTFS, quickly becoming an industry standard. Here's what makes it so advantageous:
+
+- Simple: The core GTFS data is text-based and easy to understand. 
```
I think we can make the "Simple", "Flexible", etc in bold :)

isabelle-dr commented on docs/getting_started/why_use_GTFS.md:
```diff
@@ -1,17 +1,39 @@
+As an open data standard, GTFS is driven by an open community where anyone can propose changes or improvements. The specification continues to evolve and is overseen by an independent nonprofit organization called [MobilityData](https://mobilitydata.org/) guided by the following characteristics:
 
-## GTFS is more than just trip planning
+- Simple: It’s meant to be easy to view and edit using simple tools and editors.
```
This seems a little duplicative of the beginning of the doc, could we add "Stable" and "User-centered" in the list above?

Sergiodero replied:
I'm having second thoughts about this one, I do agree that it might look a bit repetitive, but basically the idea for the first list was to set the structure for the rest of the text. As you can see, Simple, Flexible, Open-Source and Community-Driven are linked with the four subtitles of this page. 

For this list, I was presenting a simplified version of the guiding principles from the Spec Amendment Process. As a fix I updated the list to match the guiding principles more closely. LMK what you think!

isabelle-dr reviewed (changes requested):
Thanks for working on this! 
I left my comments in-line and proposed a few suggestions

Sergiodero committed:
Update docs/getting_started/create.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/publish.md

add link to resources: feed aggregators

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/publish.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/validate.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/what_is_GTFS.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/what_is_GTFS.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/what_is_GTFS.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/validate.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/validate.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Update docs/getting_started/validate.md

Co-authored-by: isabelle-dr <isabelle@mobilitydata.org>

Sergiodero committed:
Address comments 20240516

Pages updated:

- What is GTFS?
- Why use GTFS?
- Create
- Validate

Sergiodero commented:
Thanks @isabelle-dr I've addressed the comments and accepted the suggestions, there's only [one change that I left as unresolved](https://github.com/MobilityData/elrond.gtfs.org/pull/26#discussion_r1603914796), where I proposed a new text. Do you mind having a look at it? Thanks!

isabelle-dr reviewed (approved):
LGTM!
